### DISCUSSION TITLE
Enhancement to the option: "Suggest a new source"

### DISCUSSION THREAD

**Discussion by rvrlx** [dailydotdev/daily#955] [Feature Requests]
Hi,

Problem: Keeping track of new source suggestions which were submitted in the past is not possible. Currently, I am keeping track of this in a list in a file on my computer. 

Suggestions: 
1) Add a way for the user to verify and track if the source has been submitted before or not, and whether it was approved or rejected. This can be part of the pre-submission steps. 
2) Add an indicator in the browser to indicate the status of the site currently being viewed (for example: is not a source, was submitted before but rejected). Perhaps change the colour of the daily dev icon in the taskbar to indicate this status.

**Comment by idoshamun** [comment]
Hi @rvrlx,

You're right the process can be improved. Let me forward it internally :)
Thanks a lot

**Reply by rvrlx** [reply]
Thanks @idoshamun.
